### ISSUE TITLE
NeoOffice won't launch under MacOS 12.4

### ISSUE THREAD

**Issue by Jajoric** [neooffice/NeoOffice#7]
Since upgrading to MacOS 12.4 NeoOffice has stopped launching. Neither the App store version nor the Professional Edition will launch. I've tried all the troubleshooting steps here: https://neowiki.neooffice.org/index.php/NeoOffice_2022_Upgrade_Issues#Step_5_-_If_NeoOffice_crashes_or_does_not_save.2C_try_these_steps I also tried deleting everything and reinstalling 2017.33 Professional Edition and that didn't work, and then tried installing Patch 0, but got a message saying that an older version of NeoOffice was on the machine, and I needed to install 2017.33 - even though that was what was I had just installed.

MacBook Pro (13-inch, 2020, Four Thunderbolt 3 ports), 2 GHz Quad-Core Intel Core i5, macOS 12.4 (21F79)

**Comment by foss-** [comment]
Can not confirm. macOS 12.4, NO 2022 Professional Edition working as expected.

**Comment by Jajoric** [comment]
I have also gone through the delete process again, and rebooted in safe mode. I then downloaded and freshly installed NeoOffice, but with the same outcome: "The application cannot be started.
The configuration directory "/Users/jamesrichards/Library/Preferences/NeoOffice-4.0" could not be found."

**Comment by plubius** [comment]
From your last post, it sounds like macOS won't let NeoOffice create any files or folders in /Users/[username]/Library/Preferences/ for some unknown reason.

You may want to try installing NeoOffice 2022 from the Mac App Store.  The Mac App Store version creates its preferences folder in a different folder than NeoOffice 2022 Professional Edition. If you install the Mac App Store version, do you get an error that /Users/[username]/Library/Containers/org.neooffice.NeoOfficeSecureEdition/Data/Library/Preferences/NeoOfficeSecureEdition-4.0 cannot be found?

**Comment by Jajoric** [comment]
Thank you for your post which has pointed me in the right direction. I found I was also having a problem with Audacity which was also failing to launch with the message "The Audacity IPC server failed to initialize. This is likely due to a resource shortage and a reboot may be required." This along with your post pointed to some underlying problem with my system. Fixing permissions didn't seem to resolve it, so I reinstalled the OS from Recovery Mode and now NeoOffice is launching correctly (and Audacity too). I am sorry to have troubled you with what was clearly a system issue with my machine.
